index.html

```diff
@@ -13,31 +13,27 @@
     <link href="https://fonts.googleapis.com/css?family=Domine&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
     <link rel="stylesheet" href ="normalize.css">
-    <link rel="stylesheet" href="main.css">
-=======
     <link rel="stylesheet" href="styles/main.css">
->>>>>>> refs/heads/master
+
     <title>Kevin's CSD120 Website</title>
 </head>
 <body>
     <header>
         <nav>
             <ul>
-                <li class="header"><a href="index.html">Home</a></li>
-                <li class="header"><a href="links.html">Links</a></li>
-                <li class="header"><a href="glossary.html">Gallery</a></li>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="links.html">Links</a></li>
+                <li><a href="glossary.html">Gallery</a></li>
             </ul>
         </nav>
     </header>
     <main>
     <section class="bio">
-        <h1>
-            <figure class="mugshot">
-                <figcaption>Me 1 year ago</figcaption>
-                <a href="../images/mugshot.jpg><img src="mugshotthumbnail.jpg" alt="Me, Kevin!" height="100" width="100"></a>
-            </figure>
-             Welcome to Kevin's CSD120 Website!
-        </h1>
+        <h1>Welcome to Kevin's CSD120 Website!</h1>
+        <figure class="mugshot">
+            <figcaption>Me 1 year ago</figcaption>
+            <a href="images/mugshot.jpg"><img src="images/mugshotthumbnail.jpg" alt="Me, Kevin!" height="100" width="100"></a>
+        </figure>
```

Comment:
When I say 'after the <h1> element` I mean after the WHOLE element, not nested inside it.